keras/losses.py

```diff
@@ -1468,11 +1469,14 @@ def _ragged_tensor_mae(y_true, y_pred):
               'keras.losses.mean_absolute_percentage_error',
               'keras.losses.mape', 'keras.losses.MAPE')
 @tf.__internal__.dispatch.add_dispatch_support
-def mean_absolute_percentage_error(y_true, y_pred):
+def mean_absolute_percentage_error(y_true, y_pred, ignore_zeros):
   """Computes the mean absolute percentage error between `y_true` and `y_pred`.
 
   `loss = 100 * mean(abs((y_true - y_pred) / y_true), axis=-1)`
 
+  If `ignore_zeros` is set to True, the calculation ignores values with 0. Otherwise,
+  a small epsilon value is used to avoid division with 0.
+
```

Comment:
Please provide a sample use case for ignore_zeros = True (and about why user should use it).

Reply:
Please also update the docstring for the ignore_zero below.